.github/scripts/setup_spark_remote.sh

```diff
@@ -2,31 +2,36 @@
 
 set -xv
```

Comment:
Consider also adding -e to exit on any command failure

Reply:
Done